test/y2partitioner/widgets/pages/bcaches_test.rb

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env rspec
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com
+
+require_relative "../../test_helper"
+
+require "cwm/rspec"
+require "y2partitioner/widgets/pages"
+
+describe Y2Partitioner::Widgets::Pages::Bcaches do
+  before { devicegraph_stub(scenario) }
+  let(:scenario) { "bcache1.xml" }
+
+  let(:device_graph) { Y2Partitioner::DeviceGraphs.instance.current }
+
+  subject { described_class.new(bcaches, pager) }
+
+  let(:bcaches) { device_graph.bcaches }
+
+  let(:pager) { double("OverviewTreePager") }
+
+  include_examples "CWM::Page"
+
+  describe "#contents" do
+    let(:widgets) { Yast::CWM.widgets_in_contents([subject]) }
+    let(:table) { widgets.detect { |i| i.is_a?(Y2Partitioner::Widgets::BlkDevicesTable) } }
+    let(:bcaches_and_parts) do
+      device_graph.bcaches
+      # TODO: partitions does not work
```

Comment:
Is this still true? As far as I understand, `Bcache` was fixed to descend from `Partitionable`

Reply:
yes, no longer true